docs/administer/api_rbac.md

```diff
@@ -0,0 +1,40 @@
+# Navigating the API changes for RBAC
```

Comment:
I think its important to mention in the heading that these are **breaking** API changes. Maybe something like:
- Navigating the breaking API changes for RBAC
- Breaking API changes in Everest 1.2.0

Reply:
Sure

Reply:
Done